derive_builder_core/src/builder.rs

```diff
@@ -80,7 +83,7 @@ impl<'a> ToTokens for Builder<'a> {
                    impl_generics);
 
             tokens.append(quote!(
-                #[derive(Default, Clone)]
+                #[derive(Default, Clone #( , #derives)* )]
```

Comment:
I think it would be cleaner, if we move `Default` and `Clone` into the derive options - even, if we set it unconditionally.